monai/fl/client/monai_algo.py

```diff
@@ -465,6 +469,9 @@ def initialize(self, extra=None):
             self.eval_workflow.bundle_root = self.bundle_root
             if self.tracking is not None and isinstance(self.eval_workflow, ConfigWorkflow):
                 ConfigWorkflow.patch_bundle_tracking(parser=self.eval_workflow.parser, settings=settings_)
+                # set the MLFlow run name to the FL client name and timestamp
+                if self.eval_workflow.parser.get("run_name", None) is None:
+                    self.eval_workflow.parser["run_name"] = f"{self.client_name}_{time.strftime('%Y%m%d_%H%M%S')}"
```

Comment:
We should compute the timestamp once and use it in both train and eval workflows.

Reply:
Sure, updated in the latest commit.

Thanks.